lib/metrics/reporter.js

```diff
@@ -55,6 +61,23 @@ class MetricsReporter extends Reporter {
       collector.collect(next())
     }
   }
+
+  isStaleMetric (metric) {
+    // if a metric is a counting metric and that count is
+    // zero, then the metric is considered stale
+    if (metric.metricImpl && metric.metricImpl._count === 0) {
+      return true
+    }
+    return false
+  }
+
+  removeMetricFromRegistry (metric) {
```

Comment:
Unfortunate the node-measured libs are not more usable here (e.g. no `deleteMetric` on https://github.com/yaorg/node-measured/blob/master/packages/measured-reporting/lib/registries/DimensionAwareMetricsRegistry.js and no `getValue()` or something accessor for Counter._count), but that's not at all a job for this PR.

Reply:
yeah, I didn't like dipping into the private-ish APIs to accomplish this -- but the library left me with no choice.